__tests__/app.test.js

```diff
@@ -31,4 +31,14 @@ describe('CORE: GET /api/topics', () => {
         return request(app).get('/api/topics/not-a-route')
         .expect(404)
         })    
-        });
\ No newline at end of file
+        });
+
+    describe('CORE: GET /api', () => {
+        test('GET:200 responds with an object describing all available endpoints', () => {
+            return request(app).get('/api')
+            .expect(200)
+            .then((response) => {
+            expect(typeof response).toBe('object')
```

Comment:
This needs to be more specific. What do you expect the object to look like?

Consider using matchers like `.toHaveProperty` to achieve this